src/frontends/onnx/frontend/src/op/random_uniform_like.cpp

```diff
@@ -28,18 +36,14 @@ OutputVector random_uniform_like(const Node& node) {
         target_type = input.get_element_type();
     }
 
-    const auto target_shape = std::make_shared<default_opset::ShapeOf>(input);
+    auto high_convert = std::make_shared<ov::op::v0::Convert>(high_const, target_type);
```

Comment:
I don't like this part without any conditions. Could you do something like
target_type == ov::element::f32 ? high_const : std::make_shared<ov::op::v0::Convert>(high_const, target_type)
? Maybe directly in RandomUniform creation call.